---
layout: default
---

<script type="text/javascript">
<!--
// QuickSearch script for JabRef HTML export 
// Version: 3.0
//
// Copyright (c) 2006-2011, Mark Schenk
//
// This software is distributed under a Creative Commons Attribution 3.0 License
// http://creativecommons.org/licenses/by/3.0/
//
// Features:
// - intuitive find-as-you-type searching
//    ~ case insensitive
//    ~ ignore diacritics (optional)
//
// - search with/without Regular Expressions
// - match BibTeX key
//

// Search settings
var searchAbstract = true;	// search in abstract
var searchcomment = true;	// search in comment

var noSquiggles = true; 	// ignore diacritics when searching
var searchRegExp = false; 	// enable RegExp searches


if (window.addEventListener) {
	window.addEventListener("load",initSearch,false); }
else if (window.attachEvent) {
	window.attachEvent("onload", initSearch); }

function initSearch() {
	// check for quick search table and searchfield
	if (!document.getElementById('qs_table')||!document.getElementById('quicksearch')) { return; }

	// load all the rows and sort into arrays
	loadTableData();
	
	//find the query field
	qsfield = document.getElementById('qs_field');

	// previous search term; used for speed optimisation
	prevSearch = '';

	//find statistics location
	stats = document.getElementById('stat');
	setStatistics(-1);
	
	// set up preferences
	initPreferences();

	// shows the searchfield
	document.getElementById('quicksearch').style.display = 'block';
	document.getElementById('qs_field').onkeyup = quickSearch;
}

function loadTableData() {
	// find table and appropriate rows
	searchTable = document.getElementById('qs_table');
	var allRows = searchTable.getElementsByTagName('tbody')[0].getElementsByTagName('tr');

	// split all rows into entryRows and infoRows (e.g. abstract, comment, bibtex)
	entryRows = new Array(); infoRows = new Array(); absRows = new Array(); revRows = new Array();

	// get data from each row
	entryRowsData = new Array(); absRowsData = new Array(); revRowsData = new Array(); 
	
	BibTeXKeys = new Array();
	
	for (var i=0, k=0, j=0; i<allRows.length;i++) {
		if (allRows[i].className.match(/entry/)) {
			entryRows[j] = allRows[i];
			entryRowsData[j] = stripDiacritics(getTextContent(allRows[i]));
			allRows[i].id ? BibTeXKeys[j] = allRows[i].id : allRows[i].id = 'autokey_'+j;
			j ++;
		} else {
			infoRows[k++] = allRows[i];
			// check for abstract/comment
			if (allRows[i].className.match(/abstract/)) {
				absRows.push(allRows[i]);
				absRowsData[j-1] = stripDiacritics(getTextContent(allRows[i]));
			} else if (allRows[i].className.match(/comment/)) {
				revRows.push(allRows[i]);
				revRowsData[j-1] = stripDiacritics(getTextContent(allRows[i]));
			}
		}
	}
	//number of entries and rows
	numEntries = entryRows.length;
	numInfo = infoRows.length;
	numAbs = absRows.length;
	numRev = revRows.length;
}

function quickSearch(){
	
	tInput = qsfield;

	if (tInput.value.length == 0) {
		showAll();
		setStatistics(-1);
		qsfield.className = '';
		return;
	} else {
		t = stripDiacritics(tInput.value);

		if(!searchRegExp) { t = escapeRegExp(t); }
			
		// only search for valid RegExp
		try {
			textRegExp = new RegExp(t,"i");
			closeAllInfo();
			qsfield.className = '';
		}
			catch(err) {
			prevSearch = tInput.value;
			qsfield.className = 'invalidsearch';
			return;
		}
	}
	
	// count number of hits
	var hits = 0;

	// start looping through all entry rows
	for (var i = 0; cRow = entryRows[i]; i++){

		// only show search the cells if it isn't already hidden OR if the search term is getting shorter, then search all
		if(cRow.className.indexOf('noshow')==-1 || tInput.value.length <= prevSearch.length){
			var found = false; 

			if (entryRowsData[i].search(textRegExp) != -1 || BibTeXKeys[i].search(textRegExp) != -1){ 
				found = true;
			} else {
				if(searchAbstract && absRowsData[i]!=undefined) {
					if (absRowsData[i].search(textRegExp) != -1){ found=true; } 
				}
				if(searchcomment && revRowsData[i]!=undefined) {
					if (revRowsData[i].search(textRegExp) != -1){ found=true; } 
				}
			}
			
			if (found){
				cRow.className = 'entry show';
				hits++;
			} else {
				cRow.className = 'entry noshow';
			}
		}
	}

	// update statistics
	setStatistics(hits)
	
	// set previous search value
	prevSearch = tInput.value;
}


// Strip Diacritics from text
// http://stackoverflow.com/questions/990904/javascript-remove-accents-in-strings

// String containing replacement characters for stripping accents 
var stripstring = 
    'AAAAAAACEEEEIIII'+
    'DNOOOOO.OUUUUY..'+
    'aaaaaaaceeeeiiii'+
    'dnooooo.ouuuuy.y'+
    'AaAaAaCcCcCcCcDd'+
    'DdEeEeEeEeEeGgGg'+
    'GgGgHhHhIiIiIiIi'+
    'IiIiJjKkkLlLlLlL'+
    'lJlNnNnNnnNnOoOo'+
    'OoOoRrRrRrSsSsSs'+
    'SsTtTtTtUuUuUuUu'+
    'UuUuWwYyYZzZzZz.';

function stripDiacritics(str){

    if(noSquiggles==false){
        return str;
    }

    var answer='';
    for(var i=0;i<str.length;i++){
        var ch=str[i];
        var chindex=ch.charCodeAt(0)-192;   // Index of character code in the strip string
        if(chindex>=0 && chindex<stripstring.length){
            // Character is within our table, so we can strip the accent...
            var outch=stripstring.charAt(chindex);
            // ...unless it was shown as a '.'
            if(outch!='.')ch=outch;
        }
        answer+=ch;
    }
    return answer;
}

// http://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex
// NOTE: must escape every \\ in the export code because of the JabRef Export...
function escapeRegExp(str) {
  return str.replace(/[-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
}

function toggleInfo(articleid,info) {

	var entry = document.getElementById(articleid);
	var abs = document.getElementById('abs_'+articleid);
	var rev = document.getElementById('rev_'+articleid);
	var bib = document.getElementById('bib_'+articleid);
	
	if (abs && info == 'abstract') {
		abs.className.indexOf('noshow') == -1?abs.className = 'abstract noshow':abs.className = 'abstract show';
	} else if (rev && info == 'comment') {
		rev.className.indexOf('noshow') == -1?rev.className = 'comment noshow':rev.className = 'comment show';
	} else if (bib && info == 'bibtex') {
		bib.className.indexOf('noshow') == -1?bib.className = 'bibtex noshow':bib.className = 'bibtex show';
	} else { 
		return;
	}

	// check if one or the other is available
	var revshow; var absshow; var bibshow;
	(abs && abs.className.indexOf('noshow') == -1)? absshow = true: absshow = false;
	(rev && rev.className.indexOf('noshow') == -1)? revshow = true: revshow = false;	
	(bib && bib.className.indexOf('noshow') == -1)? bibshow = true: bibshow = false;
	
	// highlight original entry
	if(entry) {
		if (revshow || absshow || bibshow) {
		entry.className = 'entry highlight show';
		} else {
		entry.className = 'entry show';
		}
	}
	
	// When there's a combination of abstract/comment/bibtex showing, need to add class for correct styling
	if(absshow) {
		(revshow||bibshow)?abs.className = 'abstract nextshow':abs.className = 'abstract';
	} 
	if (revshow) {
		bibshow?rev.className = 'comment nextshow': rev.className = 'comment';
	}	
	
}

function setStatistics (hits) {
	if(hits < 0) { hits=numEntries; }
	if(stats) { stats.firstChild.data = hits + '/' + numEntries}
}

function getTextContent(node) {
	// Function written by Arve Bersvendsen
	// http://www.virtuelvis.com
	
	if (node.nodeType == 3) {
	return node.nodeValue;
	} // text node
	if (node.nodeType == 1 && node.className != "infolinks") { // element node
	var text = [];
	for (var chld = node.firstChild;chld;chld=chld.nextSibling) {
		text.push(getTextContent(chld));
	}
	return text.join("");
	} return ""; // some other node, won't contain text nodes.
}

function showAll(){
	closeAllInfo();
	for (var i = 0; i < numEntries; i++){ entryRows[i].className = 'entry show'; }
}

function closeAllInfo(){
	for (var i=0; i < numInfo; i++){
		if (infoRows[i].className.indexOf('noshow') ==-1) {
			infoRows[i].className = infoRows[i].className + ' noshow';
		}
	}
}

function clearQS() {
	qsfield.value = '';
	showAll();
}

function redoQS(){
	showAll();
	quickSearch(qsfield);
}

function updateSetting(obj){
	var option = obj.id;
	var checked = obj.value;

	switch(option)
	 {
	 case "opt_searchAbs":
	   searchAbstract=!searchAbstract;
	   redoQS();
	   break;
	 case "opt_searchRev":
	   searchcomment=!searchcomment;
	   redoQS();
	   break;
	 case "opt_useRegExp":
	   searchRegExp=!searchRegExp;
	   redoQS();
	   break;
	 case "opt_noAccents":
	   noSquiggles=!noSquiggles;
	   loadTableData();
	   redoQS();
	   break;
	 }
}

function initPreferences(){
	if(searchAbstract){document.getElementById("opt_searchAbs").checked = true;}
	if(searchcomment){document.getElementById("opt_searchRev").checked = true;}
	if(noSquiggles){document.getElementById("opt_noAccents").checked = true;}
	if(searchRegExp){document.getElementById("opt_useRegExp").checked = true;}
	
	if(numAbs==0) {document.getElementById("opt_searchAbs").parentNode.style.display = 'none';}
	if(numRev==0) {document.getElementById("opt_searchRev").parentNode.style.display = 'none';}	
}

function toggleSettings(){
	var togglebutton = document.getElementById('showsettings');
	var settings = document.getElementById('settings');
	
	if(settings.className == "hidden"){
		settings.className = "show";
		togglebutton.innerText = "close settings";
		togglebutton.textContent = "close settings";
	}else{
		settings.className = "hidden";
		togglebutton.innerText = "settings...";		
		togglebutton.textContent = "settings...";
	}
}

-->
</script>
<style type="text/css">

form#quicksearch { width: auto; border-style: solid; border-color: gray; border-width: 1px 0px; padding: 0.7em 0.5em; display:none; position:relative; }
span#searchstat {padding-left: 1em;}

div#settings { margin-top:0.7em; /* border-bottom: 1px transparent solid; background-color: #efefef; border: 1px grey solid; */ }
div#settings ul {margin: 0; padding: 0; }
div#settings li {margin: 0; padding: 0 1em 0 0; display: inline; list-style: none; }
div#settings li + li { border-left: 2px #efefef solid; padding-left: 0.5em;}
div#settings input { margin-bottom: 0px;}

div#settings.hidden {display:none;}

#showsettings { border: 1px grey solid; padding: 0 0.5em; float:right; line-height: 1.6em; text-align: right; }
#showsettings:hover { cursor: pointer; }

.invalidsearch { background-color: red; }
input[type="button"] { background-color: #efefef; border: 1px #2E2E2E solid;}

table { border: 1px gray none; width: 100%; empty-cells: show; border-spacing: 0em 0.1em; margin: 1em 0em; }
th, td { border: none; padding: 0.5em; vertical-align: top; text-align: justify; }

td a { color: navy; text-decoration: none; }
td a:hover  { text-decoration: underline; }

tr.noshow { display: none;}
tr.highlight td { background-color: #EFEFEF; border-top: 2px #2E2E2E solid; font-weight: bold; }
tr.abstract td, tr.comment td, tr.bibtex td { background-color: #EFEFEF; text-align: justify; border-bottom: 2px #2E2E2E solid; }
tr.nextshow td { border-bottom-style: none; }

tr.bibtex pre { width: 100%; overflow: auto; white-space: pre-wrap;}
p.infolinks { margin: 0.3em 0em 0em 0em; padding: 0px; }

@media print {
	p.infolinks, #qs_settings, #quicksearch, t.bibtex { display: none !important; }
	tr { page-break-inside: avoid; }
}
</style>

{% if page.header.overlay_color or page.header.overlay_image or page.header.image %}
  {% include page__hero.html %}
{% elsif page.header.video.id and page.header.video.provider %}
  {% include page__hero_video.html %}
{% endif %}

{% if page.url != "/" and site.breadcrumbs %}
  {% unless paginator %}
    {% include breadcrumbs.html %}
  {% endunless %}
{% endif %}

<div id="main" role="main">
  {% include sidebar.html %}

  <div class="archive">
    {% unless page.header.overlay_color or page.header.overlay_image %}
      <h1 class="page__title">{{ page.title }}</h1>
    {% endunless %}
    {{ content }}

  <form action="" id="quicksearch">
      <input type="text" id="qs_field" autocomplete="off" placeholder="Type to search..." /> <input type="button" onclick="clearQS()" value="clear" />
      <span id="searchstat">Matching entries: <span id="stat">0</span></span>
      <div id="showsettings" onclick="toggleSettings()">settings...</div>
      <div id="settings" class="hidden">
          <ul>
              <li><input type="checkbox" class="search_setting" id="opt_searchAbs" onchange="updateSetting(this)"><label for="opt_searchAbs"> include abstract</label></li>
              <li><input type="checkbox" class="search_setting" id="opt_searchRev" onchange="updateSetting(this)"><label for="opt_searchRev"> include comment</label></li>
              <li><input type="checkbox" class="search_setting" id="opt_useRegExp" onchange="updateSetting(this)"><label for="opt_useRegExp"> use RegExp</label></li>
              <li><input type="checkbox" class="search_setting" id="opt_noAccents" onchange="updateSetting(this)"><label for="opt_noAccents"> ignore accents</label></li>
          </ul>
      </div>
  </form>
  <!--BEGIN PUBLICATIOON INSERT HERE-->
<table id="qs_table" border="1">
<tbody>
<tr id="Benson2018" class="entry">
    <td> [1] Benson, A.R.; Abebe, R.; Schaub, M.T.; Jadbabaie, A. &amp; Kleinberg, J. (2018), <i>"Simplicial closure and higher-order link prediction"</i>, Proceedings of the National Academy of Sciences., November, 2018. 
        <p class="infolinks">[<a href="javascript:toggleInfo('Benson2018','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Benson2018','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1073/pnas.1800683115" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1802.06916" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Benson2018" class="abstract noshow">
    <td><b>Abstract</b>: Networks provide a powerful formalism for modeling complex systems by using a model of pairwise interactions. But much of the structure within these systems involves interactions that take place among more than two nodes at once&mdash;for example, communication within a group rather than person to person, collaboration among a team rather than a pair of coauthors, or biological interaction between a set of molecules rather than just two. Such higher-order interactions are ubiquitous, but their empirical study has received limited attention, and little is known about possible organizational principles of such structures. Here we study the temporal evolution of 19 datasets with explicit accounting for higher-order interactions. We show that there is a rich variety of structure in our datasets but datasets from the same system types have consistent patterns of higher-order structure. Furthermore, we find that tie strength and edge density are competing positive indicators of higher-order organization, and these trends are consistent across interactions involving differing numbers of nodes. To systematically further the study of theories for such higher-order structures, we propose higher-order link prediction as a benchmark problem to assess models and algorithms that predict higher-order structure. We find a fundamental difference from traditional pairwise link prediction, with a greater role for local rather than long-range information in predicting the appearance of new interactions.</td>
</tr>
<tr id="bib_Benson2018" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Benson2018,
    author = {Benson, Austin R. and Abebe, Rediet and Schaub, Michael T. and Jadbabaie, Ali and Kleinberg, Jon},
           title = {Simplicial closure and higher-order link prediction},
           journal = {Proceedings of the National Academy of Sciences},
           year = {2018},
           url = {https://arxiv.org/abs/1802.06916},
           doi = {10.1073/pnas.1800683115}
}
    </pre></td>
</tr>
<tr id="Schaub2018b" class="entry">
	<td> [2] Schaub MT and Segarra S (2018), <i>"Flow smoothing and denoising: graph signal processing in the edge-space"</i>. August, 2018.
	<p class="infolinks">[<a href="javascript:toggleInfo('Schaub2018b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2018b','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1808.02111" target="_blank">URL</a>]</p>
	</td>
</tr>
<tr id="abs_Schaub2018b" class="abstract noshow">
	<td><b>Abstract</b>: This paper focuses on devising graph signal processing tools for the treatment of data defined on the edges of a graph. We first show that conventional tools from graph signal processing may not be suitable for the analysis of such signals. More specifically, we discuss how the underlying notion of a `smooth signal' inherited from (the typically considered variants of) the graph Laplacian are not suitable when dealing with edge signals that encode a notion of flow. To overcome this limitation we introduce a class of filters based on the Edge-Laplacian, a special case of the Hodge-Laplacian for simplicial complexes of order one. We demonstrate how this Edge-Laplacian leads to low-pass filters that enforce (approximate) flow-conservation in the processed signals. Moreover, we show how these new filters can be combined with more classical Laplacian-based processing methods on the line-graph. Finally, we illustrate the developed tools by denoising synthetic traffic flows on the London street network. </td>
</tr>
<tr id="bib_Schaub2018b" class="bibtex noshow">
<td><b>BibTeX</b>:
<pre>
@misc{Schaub2018b,
  author = {Schaub, M. T. and Segarra, S.},
  title = {Flow smoothing and denoising: graph signal processing in the edge-space},
  year = {2018},
  note = {submitted},
  url = {https://arxiv.org/abs/1808.02111}
}
</pre></td>
</tr>
<tr id="Schaub2018c" class="entry">
	<td> [3] Schaub MT, Benson AR, Horn P, Lippner G and Jadbabaie A (2018), <i>"Random walks on simplicial complexes and the normalized Hodge Laplacian"</i>, submitted. July, 2018.
	<p class="infolinks">[<a href="javascript:toggleInfo('Schaub2018c','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2018c','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1807.05044" target="_blank">URL</a>]</p>
	</td>
</tr>
<tr id="abs_Schaub2018c" class="abstract noshow">
	<td><b>Abstract</b>: Modeling complex systems and data with graphs has been a mainstay of the applied mathematics community. The nodes in the graph represent entities and the edges model the relations between them. Simplicial complexes, a mathematical object common in topological data analysis, have emerged as a model for multi-nodal interactions that occur in several complex systems; for example, biological interactions occur between a set of molecules rather than just two, and commu- nication systems can have group messages and not just person-to-person messages. While simplicial complexes can model multi-nodal interactions, many ideas from network analysis concerning dynam- ical processes lack a proper correspondence in the a simplicial complex model. In particular, diffusion processes and random walks, which underpin large parts of the network analysis toolbox including centrality measures and ranking, community detection, and contagion models, have so far been only scarcely studied for simplicial complexes. Here we develop a diffusion process on simplicial com- plexes that can serve as a foundation for making simplicial complex models more effective. Our key idea is to generalize the well-known relationship between the normalized graph Laplacian operator and random walks on graphs by devising an appropriate normalization for the Hodge Laplacian, the analog of the graph Laplacian for simplicial complexes. We demonstrate how our diffusion process can be used for system analysis by developing a generalization of PageRank for edges in simplicial complexes and analyzing a book co-purchasing dataset.</td>
</tr>
<tr id="bib_Schaub2018c" class="bibtex noshow">
<td><b>BibTeX</b>:
<pre>
@misc{Schaub2018c,
  author = {Michael T. Schaub and Austin R. Benson and Paul Horn and Gabor Lippner and Ali Jadbabaie},
  title = {Random walks on simplicial complexes and the normalized Hodge Laplacian},
  howpublished = {submitted},
  year = {2018},
  url = {https://arxiv.org/abs/1807.05044}
}
</pre></td>
</tr>
<tr id="Schaub2018" class="entry">
	<td> [4] Schaub MT, Delvenne J-C, Lambiotte R and Barahona M (2018), <i>"Advances in network clustering and blockmodeling (forthcoming)"</i>. 
	<p class="infolinks">[<a href="javascript:toggleInfo('Schaub2018','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2018','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1804.06268" target="_blank">URL</a>]</p>
	</td>
</tr>
<tr id="abs_Schaub2018" class="abstract noshow">
	<td><b>Abstract</b>: This chapter discusses the interplay between structure and dynamics in complex networks. Given a particular network with an endowed dynamics, our goal is to find partitions aligned with the dynamical process acting on top of the network. We thus aim to gain a reduced description of the system that takes into account both its structure and dynamics.<br>In the first part, we introduce the general mathematical setup for the types of dynamics we consider throughout the chapter. We provide two guiding examples, namely consensus dynamics and diffusion processes (random walks), motivating their connection to social network analysis, and provide a brief discussion on the general dynamical framework and its possible extensions.<br>In the second part, we focus on the influence of graph structure on the dynamics taking place on the network, focussing on three concepts that allow us to gain insight into this notion. First, we describe how time scale separation can appear in the dynamics on a network as a consequence of graph structure. Second, we discuss how the presence of particular symmetries in the network give rise to invariant dynamical subspaces that can be precisely described by graph partitions. Third, we show how this dynamical viewpoint can be extended to study dynamics on networks with signed edges, which allow us to discuss connections to concepts in social network analysis, such as structural balance.<br>In the third part, we discuss how to use dynamical processes unfolding on the network to detect meaningful network substruc- tures. We then show how such dynamical measures can be related to seemingly different algorithm for community detection and coarse-graining proposed in the literature. We conclude with a brief summary and highlight interesting open future directions.<br></td>
</tr>
<tr id="bib_Schaub2018" class="bibtex noshow">
<td><b>BibTeX</b>:
<pre>
@misc{Schaub2018,
  author = {Michael T. Schaub and Jean-Charles Delvenne and Renaud Lambiotte and Mauricio Barahona},
  editor = {Patrick Doreian and Vladimir Batagelj and Anuska Ferligoj},
  title = {Advances in network clustering and blockmodeling (forthcoming)},
  publisher = {Wiley},
  year = {2018},
  pages = {(forthcoming)},
  url = {https://arxiv.org/abs/1804.06268}
}
</pre></td>
</tr>
<tr id="Schaub2018a" class="entry">
	<td> [5] Schaub MT, Delvenne J-C, Lambiotte R and Barahona M (2018), <i>"Multiscale Dynamical Embeddings of Complex Networks"</i>, submitted. April, 2018.
	<p class="infolinks">[<a href="javascript:toggleInfo('Schaub2018a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2018a','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1804.03733" target="_blank">URL</a>]</p>
	</td>
</tr>
<tr id="abs_Schaub2018a" class="abstract noshow">
	<td><b>Abstract</b>: Complex systems and relational data are often abstracted as dynamical processes on networks. To understand, predict and control their behavior a crucial step is to extract reduced descriptions of such networks. Inspired by notions from Control Theory, here we propose a time-dependent dynamical similarity measure between nodes, which quantifies the effect that a node input has on the network over time. This dynamical similarity induces an embedding that can be employed for several analysis tasks. Here we focus on (i) dimensionality reduction, by projecting nodes onto a low dimensional space capturing dynamic similarity at different time scales, and (ii) how to exploit our embeddings to uncover functional modules. We exemplify our ideas through case studies focusing on directed networks without strong connectivity, and signed networks. We further highlight how several ideas from community detection can be generalized in terms of our embedding perspective and linked to ideas from Control Theory. </td>
</tr>
<tr id="bib_Schaub2018a" class="bibtex noshow">
<td><b>BibTeX</b>:
<pre>
@misc{Schaub2018a,
  author = {Michael T. Schaub and Jean-Charles Delvenne and Renaud Lambiotte and Mauricio Barahona},
  title = {Multiscale Dynamical Embeddings of Complex Networks},
  howpublished = {submitted},
  year = {2018},
  url = {https://arxiv.org/abs/1804.03733}
}
</pre></td>
</tr>
<tr id="Billeh2018" class="entry">
    <td> [6] Billeh, Y.N. &amp; Schaub, M.T. (2018), <i>"Feedforward architectures driven by inhibitory interaction patterns"</i>, Journal of Computational Neuroscience., February, 2018.  Vol. 44(1), pp. 63-74.
        <p class="infolinks">[<a href="javascript:toggleInfo('Billeh2018','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Billeh2018','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1007/s10827-017-0669-1" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1701.04905" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Billeh2018" class="abstract noshow">
    <td><b>Abstract</b>: Directed information transmission is a paramount requirement for many social, physical, and biological systems. For neural systems, scientists have studied this problem under the paradigm of feedforward networks for decades. In most models of feedforward networks, activity is exclusively driven by excitatory neurons and the wiring patterns between them while inhibitory neurons play only a stabilizing role for the network dynamics. Motivated by recent experimental discoveries of hippocampal circuitry and the diversity of inhibitory neurons throughout the brain, here we illustrate that one can construct such networks even if the connectivity between the excitatory units in the system remains random. This is achieved by endowing inhibitory nodes with a more active role in the network. Our findings demonstrate that feedforward activity can be caused by a much broader network-architectural basis than often assumed.</td>
</tr>
<tr id="bib_Billeh2018" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Billeh2018,
    author = {Yazan N. Billeh and Michael T. Schaub},
           title = {Feedforward architectures driven by inhibitory interaction patterns},
           journal = {Journal of Computational Neuroscience},
           year = {2018},
           volume = {44},
           number = {1},
           pages = {63-74},
           url = {https://arxiv.org/abs/1701.04905},
           doi = {10.1007/s10827-017-0669-1}
}
    </pre></td>
</tr>
<tr id="Rosvall2017" class="entry">
	<td> [7] Rosvall M, Delvenne J-C, Schaub MT and Lambiotte R (2017), <i>"Advances in network clustering and blockmodeling (forthcoming)"</i>. December, 2017.
	<p class="infolinks">[<a href="javascript:toggleInfo('Rosvall2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Rosvall2017','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1712.06468" target="_blank">URL</a>]</p>
	</td>
</tr>
<tr id="abs_Rosvall2017" class="abstract noshow">
	<td><b>Abstract</b>: A precise definition of what constitutes a community in networks has remained elusive. Consequently, network scientists have compared community detection algorithms on benchmark networks with a particular form of community structure and classified them based on the mathematical techniques they employ. However, this comparison can be misleading because apparent similarities in their mathematical machinery can disguise different reasons for why we would want to employ community detection in the first place. Here we provide a focused review of these different motivations that underpin community detection. This problem-driven classification is useful in applied network science, where it is important to select an appropriate algorithm for the given purpose. Moreover, highlighting the different approaches to community detection also delineates the many lines of research and points out open directions and avenues for future research. </td>
</tr>
<tr id="bib_Rosvall2017" class="bibtex noshow">
<td><b>BibTeX</b>:
<pre>
@misc{Rosvall2017,
  author = {Martin Rosvall and Jean-Charles Delvenne and Michael T Schaub and Renaud Lambiotte},
  editor = {Patrick Doreian and Vladimir Batagelj and Anuska Ferligoj},
  title = {Advances in network clustering and blockmodeling (forthcoming)},
  publisher = {Wiley},
  year = {2017},
  url = {https://arxiv.org/abs/1712.06468}
}
</pre></td>
</tr>
<tr id="Segarra2017" class="entry">
    <td> [8] Segarra, S.; Schaub, M.T. &amp; Jadbabaie, A. (2017), <i>"Network Inference from Consensus Dynamics"</i>, 56th IEEE Conference on Decision and Control (CDC 2017)., June, 2017. , pp. 3212-3217.
        <p class="infolinks">[<a href="javascript:toggleInfo('Segarra2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Segarra2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1109/CDC.2017.8264130" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1708.05329" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Segarra2017" class="abstract noshow">
    <td><b>Abstract</b>: We consider the problem of identifying the topology of a weighted, undirected network G from observing snapshots of multiple independent consensus dynamics. Specifically, we observe the opinion profiles of a group of agents for a set of M independent topics and our goal is to recover the precise relationships between the agents, as specified by the unknown network G. In order to overcome the under- determinacy of the problem at hand, we leverage concepts from spectral graph theory and convex optimization to unveil the underlying network structure. More precisely, we formulate the network inference problem as a convex optimization that seeks to endow the network with certain desired properties – such as sparsity – while being consistent with the spectral information extracted from the observed opinions. This is complemented with theoretical results proving consistency as the number M of topics grows large. We further illustrate our method by numerical experiments, which showcase the effectiveness of the technique in recovering synthetic and real-world networks.</td>
</tr>
<tr id="bib_Segarra2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Segarra2017,
    author = {Santiago Segarra and Michael T. Schaub and Ali Jadbabaie},
           title = {Network Inference from Consensus Dynamics},
           journal = {56th IEEE Conference on Decision and Control (CDC 2017)},
           year = {2017},
           pages = {3212-3217},
           url = {https://arxiv.org/abs/1708.05329},
           doi = {10.1109/CDC.2017.8264130}
}
    </pre></td>
</tr>
<tr id="Faccin2017" class="entry">
    <td> [9] Faccin, M.; Schaub, M.T. &amp; Delvenne, J.-C. (2017), <i>"Entrograms and coarse graining of dynamics on complex networks"</i>, Journal of Complex Networks., November, 2017. , pp. cnx055.
        <p class="infolinks">[<a href="javascript:toggleInfo('Faccin2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Faccin2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1093/comnet/cnx055" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1711.01987" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Faccin2017" class="abstract noshow">
    <td><b>Abstract</b>: Using an information theoretic point of view, we investigate how a dynamics acting on a network can be coarse grained through the use of graph partitions. Specifically, we are interested in how aggregating the state space of a Markov process according to a partition impacts on the thus obtained lower-dimensional dynamics. We highlight that for a dynamics on a particular graph there may be multiple coarse grained descriptions that capture different, incomparable features of the original process. For instance, a coarse graining induced by one partition may be commensurate with a time-scale separation in the dynamics, while another coarse graining may correspond to a different lower-dimensional dynamics that preserves the Markov property of the original process. Taking inspiration from the literature of Computational Mechanics, we find that a convenient tool to summarise and visualise such dynamical properties of a coarse grained model (partition) is the entrogram. The entrogram gathers certain information-theoretic measures, which quantify how information flows across time steps. These information theoretic quantities include the entropy rate, as well as a measure for the memory contained in the process, i.e., how well the dynamics can be approximated by a first order Markov process. We use the entrogram to investigate how specific macro-scale connection patterns in the state-space transition graph of the original dynamics result in desirable properties of coarse grained descriptions. We thereby provide a fresh perspective on the interplay between structure and dynamics in networks, and the process of partitioning from an information theoretic perspective. We focus on networks that may be approximated by both a core-periphery or a clustered organization, and highlight that each of these coarse grained descriptions can capture different aspects of a Markov process acting on the network.</td>
</tr>
<tr id="bib_Faccin2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Faccin2017,
    author = {Faccin, Mauro and Schaub, Michael T and Delvenne, Jean-Charles},
           title = {Entrograms and coarse graining of dynamics on complex networks},
           journal = {Journal of Complex Networks},
           year = {2017},
           pages = {cnx055},
           url = {https://arxiv.org/abs/1711.01987},
           doi = {10.1093/comnet/cnx055}
}
    </pre></td>
</tr>
<tr id="Avella-Medina2017" class="entry">
    <td> [10] Avella-Medina, M.; Parise, F.; Schaub, M.T. &amp; Segarra, S. (2017), <i>"Centrality Measures for Graphons"</i> 
        <p class="infolinks">[<a href="javascript:toggleInfo('Avella-Medina2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Avella-Medina2017','bibtex')">BibTeX</a>] [<a href="https://arxiv.org/abs/1707.09350" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Avella-Medina2017" class="abstract noshow">
    <td><b>Abstract</b>: Graphs provide a natural mathematical abstraction for systems with pairwise interactions, and thus have become a prevalent tool for the representation of systems across various scientific domains. However, as the size of relational datasets continues to grow, traditional graph-based approaches are increasingly replaced by other modeling paradigms, which enable a more flexible treatment of such datasets. A promising framework in this context is provided by graphons, which have been formally introduced as the natural limiting objects for graphs of increasing sizes. However, while the theory of graphons is already well developed, some prominent tools in network analysis still have no counterpart within the realm of graphons. In particular, node centrality measures, which have been successfully employed in various applications to reveal important nodes in a network, have so far not been defined for graphons. In this work we introduce formal definitions of centrality measures for graphons and establish their connections to centrality measures defined on finite graphs. In particular, we build on the theory of linear integral operators to define degree, eigenvector, and Katz centrality functions for graphons. We further establish concentration inequalities showing that these centrality functions are natural limits of their analogous counterparts defined on sequences of random graphs of increasing size. We discuss several strategies for computing these centrality measures, and illustrate them through a set of numerical examples.</td>
</tr>
<tr id="bib_Avella-Medina2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Avella-Medina2017,
    author = {Marco Avella-Medina and Francesca Parise and Michael T Schaub and Santiago Segarra},
           title = {Centrality Measures for Graphons},
           year = {2017},
           url = {https://arxiv.org/abs/1707.09350}
}
    </pre></td>
</tr>
<tr id="Schaub*2017" class="entry">
    <td> [11] Schaub*, M.T.; Trefois*, M.; Dooren, P.V. &amp; Delvenne, J.-C. (2017), <i>"Sparse Matrix Factorizations For Fast Iterative Linear Solvers With Application To Laplacian Systems"</i>, SIAM Journal on Matrix Analysis and Applications.  Vol. 38(2), pp. 505-529.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub*2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub*2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1137/16M1077398" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1605.09148" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub*2017" class="abstract noshow">
    <td><b>Abstract</b>: In solving a linear system with iterative methods, one is usually confronted with the dilemma of having to choose between cheap, inefficient iterates over sparse search directions (e.g., coordinate descent), or expensive iterates in well-chosen search directions (e.g., conjugate gradients). In this paper, we propose to interpolate between these two extremes, and show how to perform cheap iterations along non-sparse search directions, provided that these directions admit a new kind of sparse factorization. For example, if the search directions are the columns of a hierarchical matrix, then the cost of each iteration is typically logarithmic in the number of variables. Using some graph-theoretical results on low-stretch spanning trees, we deduce as a special case a nearly-linear time algorithm to approximate the minimal norm solution of a linear system Bx=b where B is the incidence matrix of a graph. We thereby can connect our results to recently proposed nearly-linear time solvers for Laplacian systems, which emerge here as a particular application of our sparse matrix factorization.</td>
</tr>
<tr id="bib_Schaub*2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub*2017,
    author = {Michael T. Schaub* and Maguy Trefois* and Paul Van Dooren and Jean-Charles Delvenne},
           title = {Sparse Matrix Factorizations For Fast Iterative Linear Solvers With Application To Laplacian Systems},
           journal = {SIAM Journal on Matrix Analysis and Applications},
           year = {2017},
           volume = {38},
           number = {2},
           pages = {505-529},
           url = {http://arxiv.org/abs/1605.09148},
           doi = {10.1137/16M1077398}
}
    </pre></td>
</tr>
<tr id="Kiselev2017" class="entry">
    <td> [12] Kiselev, V.Y.; Kirschner, K.; Schaub, M.T.; Andrews, T.; Chandra, T.; Natarajan, K.N.; Reik, W.; Barahona, M.; Green, A.R. &amp; Hemberg, M. (2017), <i>"SC3 - consensus clustering of single-cell RNA-Seq data"</i>, Nature Methods., March, 2017.  Vol. 14, pp. 483-486.
        <p class="infolinks">[<a href="javascript:toggleInfo('Kiselev2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kiselev2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1038/nmeth.4236" target="_blank">DOI</a>] [<a href="http://dx.doi.org/10.1101/036558" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Kiselev2017" class="abstract noshow">
    <td><b>Abstract</b>: Using single-cell RNA-seq (scRNA-seq), the full transcriptome of individual cells can be acquired, enabling a quantitative characterisation of cell-type based on expression profiles. Due to the large variability in gene expression, assigning cells into groups based on the transcriptome remains challenging. We present Single-Cell Consensus Clustering (SC3), a tool for unsupervised clustering of scRNA-seq data. SC3 integrates many different clustering solutions through a consensus approach, thereby increasing its accuracy and robustness against noise. Tests on nine published datasets show that SC3 outperforms existing methods, yet SC3 remains scalable for large datasets, as shown by the analysis of a dataset containing &nbsp;45,000 cells. To enhance the accessibility to users with limited bioinformatics expertise, SC3 features an interactive graphical implementation, which aids the biological interpretation by identifying marker genes, differentially expressed genes and outlier cells. Finally, we apply SC3 to identify different subclones of neoplastic cells in data collected from patients.</td>
</tr>
<tr id="bib_Kiselev2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Kiselev2017,
    author = {Kiselev, Vladimir Yu. and Kirschner, Kristina and Schaub, Michael T. and Andrews, Tallulah and Chandra, Tamir and Natarajan, Kedar N and Reik, Wolf and Barahona, Mauricio and Green, Anthony R and Hemberg, Martin},
           title = {SC3 - consensus clustering of single-cell RNA-Seq data},
           journal = {Nature Methods},
           year = {2017},
           volume = {14},
           pages = {483-486},
           url = {http://dx.doi.org/10.1101/036558},
           doi = {10.1038/nmeth.4236}
}
    </pre></td>
</tr>
<tr id="Schaub2017" class="entry">
    <td> [13] Schaub, M.T.; Delvenne, J.-C.; Rosvall, M. &amp; Lambiotte, R. (2017), <i>"The many facets of community detection in complex networks"</i>, Applied Network Science.  Vol. 2(1), pp. 4.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1007/s41109-017-0023-6" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1611.07769" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2017" class="abstract noshow">
    <td><b>Abstract</b>: Community detection, the decomposition of a graph into essential building blocks, has been a core research topic in network science over the past years. Since a precise notion of what constitutes a community has remained evasive, community detection algorithms have often been compared on benchmark graphs with a particular form of assortative community structure and classified based on the mathematical techniques they employ. However, this comparison can be misleading because apparent similarities in their mathematical machinery can disguise different goals and reasons for why we want to employ community detection in the first place. Here we provide a focused review of these different motivations that underpin community detection. This problem-driven classification is useful in applied network science, where it is important to select an appropriate algorithm for the given purpose. Moreover, highlighting the different facets of community detection also delineates the many lines of research and points out open directions and avenues for future research.</td>
</tr>
<tr id="bib_Schaub2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2017,
    author = {Schaub, Michael T. and Delvenne, Jean-Charles and Rosvall, Martin and Lambiotte, Renaud},
           title = {The many facets of community detection in complex networks},
           journal = {Applied Network Science},
           year = {2017},
           volume = {2},
           number = {1},
           pages = {4},
           url = {https://arxiv.org/abs/1611.07769},
           doi = {10.1007/s41109-017-0023-6}
}
    </pre></td>
</tr>
<tr id="Estrada2017" class="entry">
    <td> [14] Estrada, E.; Delvenne, J.-C.; Hatano, N.; Mateos, J.L.; Metzler, R.; Riascos, A.P. &amp; Schaub, M.T. (2017), <i>"Random Multi-Hopper Model. Super-Fast Random Walks on Graphs"</i>, Journal of Complex Networks., October, 2017. , pp. cnx043.
        <p class="infolinks">[<a href="javascript:toggleInfo('Estrada2017','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Estrada2017','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1093/comnet/cnx043" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1612.08631" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Estrada2017" class="abstract noshow">
    <td><b>Abstract</b>: We develop a model for a random walker with long-range hops on general graphs. This random multi-hopper jumps from a node to any other node in the graph with a probability that decays as a function of the shortest-path distance between the two nodes. We consider here two decaying functions in the form of the Laplace and Mellin transforms of the shortest-path distances. Remarkably, when the parameters of these transforms approach zero asymptotically, the multi-hopper's hitting times between any two nodes in the graph converge to their minimum possible value, given by the hitting times of a normal random walker on a complete graph. Stated differently, for small parameter values the multi-hopper explores a general graph as fast as possible when compared to a random walker on a full graph. Using computational experiments we show that compared to the normal random walker, the multi-hopper indeed explores graphs with clusters or skewed degree distributions more efficiently for a large parameter range. We provide further computational evidence of the speed-up attained by the random multi-hopper model with respect to the normal random walker by studying deterministic, random and real-world networks.</td>
</tr>
<tr id="bib_Estrada2017" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Estrada2017,
    author = {Estrada, E. and Delvenne, J.-C. and Hatano, N. and Mateos, J.&nbsp;L. and Metzler, R. and Riascos, A.&nbsp;P. and Schaub, M.&nbsp;T.},
           title = {Random Multi-Hopper Model. Super-Fast Random Walks on Graphs},
           journal = {Journal of Complex Networks},
           year = {2017},
           pages = {cnx043},
           url = {https://arxiv.org/abs/1612.08631},
           doi = {10.1093/comnet/cnx043}
}
    </pre></td>
</tr>
<tr id="Amor2016" class="entry">
    <td> [15] Amor, B.R.C.; Schaub, M.T.; Yaliraki, S.N. &amp; Barahona, M. (2016), <i>"Prediction of allosteric sites and mediating interactions through bond-to-bond propensities"</i>, Nature Communications., August, 2016.  Vol. 7(12477)
        <p class="infolinks">[<a href="javascript:toggleInfo('Amor2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Amor2016','comment')">Comment</a>]  [<a href="javascript:toggleInfo('Amor2016','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1038/ncomms12477" target="_blank">DOI</a>] [<a href="http://dx.doi.org/10.1101/056275" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Amor2016" class="abstract noshow">
    <td><b>Abstract</b>: Allostery is a fundamental mechanism of biological regulation, in which binding of a molecule at a distant location affects the active site of a protein. Allosteric sites provide targets to fine-tune protein activity, yet we lack computational methodologies to predict them. Here we present an efficient graph-theoretical framework to reveal allosteric interactions (atoms and communication pathways strongly coupled to the active site) without a priori information of their location. Using an atomistic graph with energy-weighted covalent and weak bonds, we define a bond-to-bond propensity quantifying the non-local effect of instantaneous bond fluctuations propagating through the protein. Significant interactions are then identified using quantile regression. We exemplify our method with three biologically important proteins: caspase-1, CheY, and h-Ras, correctly predicting key allosteric interactions, whose significance is additionally confirmed against a reference set of 100 proteins. The almost-linear scaling of our method renders it suitable for high-throughput searches for candidate allosteric sites.</td>
</tr>
<tr id="rev_Amor2016" class="comment noshow">
    <td><b>Comment</b>: News articles:<br>http://www3.imperial.ac.uk/newsandeventspggrp/imperialcollege/newssummary/news_25-8-2016-15-3-8<br>http://phys.org/news/2016-08-paths-proteins-reveal-drug.html<br>http://naturalsciencenews.com/2016/08/27/mathematical-models-used-to-study-traffic-jams-can-help-researchers-understand-proteins/?platform=hootsuite<br></td>
</tr>
<tr id="bib_Amor2016" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Amor2016,
    author = {Benjamin R. C. Amor and Michael T. Schaub and Sophia N. Yaliraki and Mauricio Barahona},
           title = {Prediction of allosteric sites and mediating interactions through bond-to-bond propensities},
           journal = {Nature Communications},
           year = {2016},
           volume = {7},
           number = {12477},
           url = {http://dx.doi.org/10.1101/056275},
           doi = {10.1038/ncomms12477}
}
    </pre></td>
</tr>
<tr id="Schaub2016" class="entry">
    <td> [16] Schaub, M.; O'Clery, N.; Billeh, Y.N.; Delvenne, J.-C.; Lambiotte, R. &amp; Barahona, M. (2016), <i>"Graph partitions and cluster synchronization in networks of oscillators"</i>, Chaos., August, 2016.  Vol. 26(9), pp. 094821.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2016','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1063/1.4961065" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/1608.04283" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2016" class="abstract noshow">
    <td><b>Abstract</b>: Synchronization dynamics depends strongly on the structure of the coupling between the oscillators. When the coupling network presents certain regularities, the dynamics can be coarse-grained into clusters by means of External Equitable Partitions and their associated quotient graphs. We exploit this graph-theoretical concept to study the phenomenon of cluster synchronization, in which different groups of nodes converge to distinct behaviors. We derive conditions and properties of networks in which such clustered behavior emerges, and show that the ensuing dynamics is the result of the localization of the eigenvectors of the associated graph Laplacians linked to the existence of invariant subspaces. The framework is applied to both linear and non- linear models, first for the standard case of networks with positive edges, before being generalized to the case of signed networks with both positive and negative interactions. We illustrate our results with examples of both signed and unsigned graphs for consensus dynamics and for partial synchronization of oscillator networks under the master stability function as well as Kuramoto oscillators.</td>
</tr>
<tr id="bib_Schaub2016" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2016,
    author = {Michael Schaub and Neave O'Clery and Yazan N. Billeh and Jean-Charles Delvenne and Renaud Lambiotte and Mauricio Barahona},
           title = {Graph partitions and cluster synchronization in networks of oscillators},
           journal = {Chaos},
           year = {2016},
           volume = {26},
           number = {9},
           pages = {094821},
           url = {https://arxiv.org/abs/1608.04283},
           doi = {10.1063/1.4961065}
}
    </pre></td>
</tr>
<tr id="Bacik2016" class="entry">
    <td> [17] Bacik, K.A.; Schaub, M.T.; Beguerisse-Diaz, M.; Billeh, Y.N. &amp; Barahona, M. (2016), <i>"Flow-Based Network Analysis of the Caenorhabditis elegans Connectome"</i>, PLoS Computational Biology., August, 2016.  Vol. 12(8), pp. 1-27.
        <p class="infolinks">[<a href="javascript:toggleInfo('Bacik2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bacik2016','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pcbi.1005055" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1511.00673" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Bacik2016" class="abstract noshow">
    <td><b>Abstract</b>: We exploit flow propagation on the directed neuronal network of the nematode C. elegans to reveal dynamically relevant features of its connectome. We find flow-based groupings of neurons at different levels of granularity, which we relate to functional and anatomical con- stituents of its nervous system. A systematic in silico evaluation of the full set of single and double neuron ablations is used to identify deletions that induce the most severe disruptions of the multi-resolution flow structure. Such ablations are linked to functionally relevant neu- rons, and suggest potential candidates for further in vivo investigation. In addition, we use the directional patterns of incoming and outgoing network flows at all scales to identify flow profiles for the neurons in the connectome, without pre-imposing a priori categories. The four flow roles identified are linked to signal propagation motivated by biological input- response scenarios.</td>
</tr>
<tr id="bib_Bacik2016" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Bacik2016,
    author = {Bacik, Karol A. AND Schaub, Michael T. AND Beguerisse-Diaz, Mariano AND Billeh, Yazan N. AND Barahona, Mauricio},
           title = {Flow-Based Network Analysis of the Caenorhabditis elegans Connectome},
           journal = {PLoS Computational Biology},
           year = {2016},
           volume = {12},
           number = {8},
           pages = {1-27},
           url = {http://arxiv.org/abs/1511.00673},
           doi = {10.1371/journal.pcbi.1005055}
}
    </pre></td>
</tr>
<tr id="Salnikov2016" class="entry">
    <td> [18] Salnikov, V.; Schaub, M.T. &amp; Lambiotte, R. (2016), <i>"Using higher-order Markov models to reveal flow-based communities in networks"</i>, Scientific Reports., March, 2016.  Vol. 6, pp. 23194.
        <p class="infolinks">[<a href="javascript:toggleInfo('Salnikov2016','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Salnikov2016','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1038/srep23194" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1601.03516" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Salnikov2016" class="abstract noshow">
    <td><b>Abstract</b>: Complex systems made of interacting elements are commonly abstracted as networks, in which nodes are associated with dynamic state variables, whose evolution is driven by interactions mediated by the edges. Markov processes have been the prevailing paradigm to model such a network-based dynamics, for instance in the form of random walks or other types of diffusions. Despite the success of this modelling perspective for numerous applications, it represents an over-simplification of several real-world systems. Importantly, simple Markov models lack memory in their dynamics, an assumption often not realistic in practice. Here, we explore possibilities to enrich the system description by means of second-order Markov models, exploiting empirical pathway information. We focus on the problem of community detection and show that standard network algorithms can be generalized in order to extract novel temporal information about the system under investigation. We also apply our methodology to temporal networks, where we can uncover communities shaped by the temporal correlations in the system. Finally, we discuss relations of the framework of second order Markov processes and the recently proposed formalism of using non-backtracking matrices for community detection.</td>
</tr>
<tr id="bib_Salnikov2016" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Salnikov2016,
    author = {Vsevolod Salnikov and Michael T. Schaub and Renaud Lambiotte},
           title = {Using higher-order Markov models to reveal flow-based communities in networks},
           journal = {Scientific Reports},
           year = {2016},
           volume = {6},
           pages = {23194},
           url = {http://arxiv.org/abs/1601.03516},
           doi = {10.1038/srep23194}
}
    </pre></td>
</tr>
<tr id="Schaub*2015" class="entry">
    <td> [19] Schaub*, M.T.; Billeh*, Y.; Anastassiou, C.A.; Koch, C. &amp; Barahona, M. (2015), <i>"Emergence of slow-switching assemblies in structured neuronal networks"</i>, PLoS Computational Biology., July, 2015.  Vol. 11(7), pp. e1004196.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub*2015','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub*2015','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pcbi.1004196" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1502.05656" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub*2015" class="abstract noshow">
    <td><b>Abstract</b>: Unraveling the interplay between connectivity and spatio-temporal dynamics in neuronal networks is a key step to advance our understanding of neuronal information processing. Here we investigate how particular features of network connectivity underpin the propensity of neural networks to generate slow-switching assembly (SSA) dynamics, i.e., sustained epochs of increased firing within assemblies of neurons which transition slowly between dif- ferent assemblies throughout the network. We show that the emergence of SSA activity is linked to spectral properties of the asymmetric synaptic weight matrix. In particular, the lead- ing eigenvalues that dictate the slow dynamics exhibit a gap with respect to the bulk of the spectrum, and the associated Schur vectors exhibit a measure of block-localization on groups of neurons, thus resulting in coherent dynamical activity on those groups. Through simple rate models, we gain analytical understanding of the origin and importance of the spectral gap, and use these insights to develop new network topologies with alternative connectivity paradigms which also display SSA activity. Specifically, SSA dynamics involv- ing excitatory and inhibitory neurons can be achieved by modifying the connectivity patterns between both types of neurons. We also show that SSA activity can occur at multiple time- scales reflecting a hierarchy in the connectivity, and demonstrate the emergence of SSA in small-world like networks. Our work provides a step towards understanding how network structure (uncovered through advancements in neuroanatomy and connectomics) can im- pact on spatio-temporal neural activity and constrain the resulting dynamics.</td>
</tr>
<tr id="bib_Schaub*2015" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub*2015,
    author = {Schaub*, Michael T. and Yazan Billeh* and Costas A. Anastassiou and Christof Koch and Mauricio Barahona},
           title = {Emergence of slow-switching assemblies in structured neuronal networks},
           journal = {PLoS Computational Biology},
           year = {2015},
           volume = {11},
           number = {7},
           pages = {e1004196},
           url = {http://arxiv.org/abs/1502.05656},
           doi = {10.1371/journal.pcbi.1004196}
}
    </pre></td>
</tr>
<tr id="Billeh*2014" class="entry">
    <td> [20] Billeh*, Y.N.; Schaub*, M.T.; Anastassiou, C.A.; Barahona, M. &amp; Koch, C. (2014), <i>"Revealing cell assemblies at multiple levels of granularity"</i>, Journal of Neuroscience Methods., Oct, 2014.  Vol. 236(0), pp. 92 - 106.
        <p class="infolinks">[<a href="javascript:toggleInfo('Billeh*2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Billeh*2014','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1016/j.jneumeth.2014.08.011" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1411.2103" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Billeh*2014" class="abstract noshow">
    <td><b>Abstract</b>: Background Current neuronal monitoring techniques, such as calcium imaging and multi-electrode arrays, enable recordings of spiking activity from hundreds of neurons simultaneously. Of primary importance in systems neuroscience is the identification of cell assemblies: groups of neurons that cooperate in some form within the recorded population. New method We introduce a simple, integrated framework for the detection of cell-assemblies from spiking data without a priori assumptions about the size or number of groups present. We define a biophysically-inspired measure to extract a directed functional connectivity matrix between both excitatory and inhibitory neurons based on their spiking history. The resulting network representation is analyzed using the Markov Stability framework, a graph theoretical method for community detection across scales, to reveal groups of neurons that are significantly related in the recorded time-series at different levels of granularity. Results and comparison with existing methods Using synthetic spike-trains, including simulated data from leaky-integrate-and-fire networks, our method is able to identify important patterns in the data such as hierarchical structure that are missed by other standard methods. We further apply the method to experimental data from retinal ganglion cells of mouse and salamander, in which we identify cell-groups that correspond to known functional types, and to hippocampal recordings from rats exploring a linear track, where we detect place cells with high fidelity. Conclusions We present a versatile method to detect neural assemblies in spiking data applicable across a spectrum of relevant scales that contributes to understanding spatio-temporal information gathered from systems neuroscience experiments.</td>
</tr>
<tr id="bib_Billeh*2014" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Billeh*2014,
    author = {Yazan N. Billeh* and Michael T. Schaub* and Costas A. Anastassiou and Mauricio Barahona and Christof Koch},
           title = {Revealing cell assemblies at multiple levels of granularity},
           journal = {Journal of Neuroscience Methods},
           year = {2014},
           volume = {236},
           number = {0},
           pages = {92 - 106},
           url = {http://arxiv.org/abs/1411.2103},
           doi = {10.1016/j.jneumeth.2014.08.011}
}
    </pre></td>
</tr>
<tr id="Schaub2014" class="entry">
    <td> [21] Schaub, M.T.; Lehmann, J.; Yaliraki, S.N. &amp; Barahona, M. (2014), <i>"Structure of complex networks: Quantifying edge-to-edge relations by failure-induced flow redistribution"</i>, Network Science., April, 2014.  Vol. 2(1), pp. 66-89.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2014','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2014','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1017/nws.2014.4" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1303.6241" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2014" class="abstract noshow">
    <td><b>Abstract</b>: The analysis of complex networks has so far revolved mainly around the role of nodes and communities of nodes. However, the dynamics of interconnected systems is often focalized on edge processes, and a dual edge-centric perspective can often prove more natural. Here we present graph-theoretical measures to quantify edge-to-edge relations inspired by the notion of flow redistribution induced by edge failures. Our measures, which are related to the pseudo-inverse of the Laplacian of the network, are global and reveal the dynamical interplay between the edges of a network, including potentially non-local interactions. Our framework also allows us to define the embeddedness of an edge, a measure of how strongly an edge features in the weighted cuts of the network. We showcase the general applicability of our edge-centric framework through analyses of the Iberian power grid, traffic flow in road networks, and the C. elegans neuronal network.</td>
</tr>
<tr id="bib_Schaub2014" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2014,
    author = {Schaub, Michael T. and Lehmann, Jörg and Yaliraki, Sophia N. and Barahona, Mauricio},
           title = {Structure of complex networks: Quantifying edge-to-edge relations by failure-induced flow redistribution},
           journal = {Network Science},
           year = {2014},
           volume = {2},
           number = {1},
           pages = {66--89},
           url = {http://arxiv.org/abs/1303.6241},
           doi = {10.1017/nws.2014.4}
}
    </pre></td>
</tr>
<tr id="Delvenne2013" class="entry">
    <td> [22] Delvenne, J.-C.; Schaub, M.T.; Yaliraki, S.N. &amp; Barahona, M. (2013), <i>"The Stability of a Graph Partition: A Dynamics-Based Framework for Community Detection"</i>, In Dynamics On and Of Complex Networks, Volume 2., May, 2013. , pp. 221-242. Springer New York.
        <p class="infolinks"> [<a href="javascript:toggleInfo('Delvenne2013','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1007/978-1-4614-6729-8_11" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1308.1605" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="bib_Delvenne2013" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@incollection{Delvenne2013,
    author = {Delvenne, Jean-Charles and Schaub, Michael T. and Yaliraki, Sophia N. and Barahona, Mauricio},
           editor = {Mukherjee, Animesh and Choudhury, Monojit and Peruani, Fernando and Ganguly, Niloy and Mitra, Bivas},
           title = {The Stability of a Graph Partition: A Dynamics-Based Framework for Community Detection},
           booktitle = {Dynamics On and Of Complex Networks, Volume 2},
           publisher = {Springer New York},
           year = {2013},
           pages = {221-242},
           url = {http://arxiv.org/abs/1308.1605},
           doi = {10.1007/978-1-4614-6729-8_11}
}
    </pre></td>
</tr>
<tr id="Schaub2012b" class="entry">
    <td> [23] Schaub, M.T.; Lambiotte, R. &amp; Barahona, M. (2012), <i>"Encoding dynamics for multiscale community detection: Markov time sweeping for the map equation"</i>, Phys. Rev. E., Aug, 2012.  Vol. 86, pp. 026112. American Physical Society.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2012b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2012b','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1103/PhysRevE.86.026112" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1109.6642" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2012b" class="abstract noshow">
    <td><b>Abstract</b>: The detection of community structure in networks is intimately related to finding a concise description of the network in terms of its modules. This notion has been recently exploited by the map equation formalism [ Rosvall and Bergstrom Proc. Natl. Acad. Sci. USA 105 1118 (2008)] through an information-theoretic description of the process of coding inter- and intracommunity transitions of a random walker in the network at stationarity. However, a thorough study of the relationship between the full Markov dynamics and the coding mechanism is still lacking. We show here that the original map coding scheme, which is both block-averaged and one-step, neglects the internal structure of the communities and introduces an upper scale, the “field-of-view” limit, in the communities it can detect. As a consequence, map is well tuned to detect clique-like communities but can lead to undesirable overpartitioning when communities are far from clique-like. We show that a signature of this behavior is a large compression gap: The map description length is far from its ideal limit. To address this issue, we propose a simple dynamic approach that introduces time explicitly into the map coding through the analysis of the weighted adjacency matrix of the time-dependent multistep transition matrix of the Markov process. The resulting Markov time sweeping induces a dynamical zooming across scales that can reveal (potentially multiscale) community structure above the field-of-view limit, with the relevant partitions indicated by a small compression gap.</td>
</tr>
<tr id="bib_Schaub2012b" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2012b,
    author = {Schaub, Michael T. and Lambiotte, Renaud and Barahona, Mauricio},
           title = {Encoding dynamics for multiscale community detection: Markov time sweeping for the map equation},
           journal = {Phys. Rev. E},
           publisher = {American Physical Society},
           year = {2012},
           volume = {86},
           pages = {026112},
           url = {http://arxiv.org/abs/1109.6642},
           doi = {10.1103/PhysRevE.86.026112}
}
    </pre></td>
</tr>
<tr id="Schaub2012" class="entry">
    <td> [24] Schaub, M.T.; Delvenne, J.-C.; Yaliraki, S.N. &amp; Barahona, M. (2012), <i>"Markov Dynamics as a Zooming Lens for Multiscale Community Detection: Non Clique-Like Communities and the Field-of-View Limit"</i>, PLoS ONE., Feb, 2012.  Vol. 7(2), pp. e32210. Public Library of Science.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2012','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2012','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pone.0032210" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1109.5593" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2012" class="abstract noshow">
    <td><b>Abstract</b>: In recent years, there has been a surge of interest in community detection algorithms for complex networks. A variety of computational heuristics, some with a long history, have been proposed for the identification of communities or, alternatively, of good graph partitions. In most cases, the algorithms maximize a particular objective function, thereby finding the ‘right’ split into communities. Although a thorough comparison of algorithms is still lacking, there has been an effort to design benchmarks, i.e., random graph models with known community structure against which algorithms can be evaluated. However, popular community detection methods and benchmarks normally assume an implicit notion of community based on clique-like subgraphs, a form of community structure that is not always characteristic of real networks. Specifically, networks that emerge from geometric constraints can have natural non clique-like substructures with large effective diameters, which can be interpreted as long-range communities. In this work, we show that long-range communities escape detection by popular methods, which are blinded by a restricted ‘field-of-view’ limit, an intrinsic upper scale on the communities they can detect. The field-of-view limit means that long-range communities tend to be overpartitioned. We show how by adopting a dynamical perspective towards community detection, in which the evolution of a Markov process on the graph is used as a zooming lens over the structure of the network at all scales, one can detect both clique- or non clique-like communities without imposing an upper scale to the detection. Consequently, the performance of algorithms on inherently low-diameter, clique-like benchmarks may not always be indicative of equally good results in real networks with local, sparser connectivity. We illustrate our ideas with constructive examples and through the analysis of real-world networks from imaging, protein structures and the power grid, where a multiscale structure of non clique-like communities is revealed.</td>
</tr>
<tr id="bib_Schaub2012" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2012,
    author = {Schaub, Michael T. AND Delvenne, Jean-Charles AND Yaliraki, Sophia N. AND Barahona, Mauricio},
           title = {Markov Dynamics as a Zooming Lens for Multiscale Community Detection: Non Clique-Like Communities and the Field-of-View Limit},
           journal = {PLoS ONE},
           publisher = {Public Library of Science},
           year = {2012},
           volume = {7},
           number = {2},
           pages = {e32210},
           url = {http://arxiv.org/abs/1109.5593},
           doi = {10.1371/journal.pone.0032210}
}
    </pre></td>
</tr>
<tr id="Schaub2012a" class="entry">
    <td> [25] Schaub, M. &amp; Schultz, S. (2012), <i>"The Ising decoder: reading out the activity of large neural ensembles"</i>, Journal of Computational Neuroscience., Feb, 2012.  Vol. 32, pp. 101-118.
        <p class="infolinks">[<a href="javascript:toggleInfo('Schaub2012a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Schaub2012a','bibtex')">BibTeX</a>] [<a href="https://doi.org/10.1007/s10827-011-0342-z" target="_blank">DOI</a>] [<a href="http://arxiv.org/abs/1009.1828" target="_blank">URL</a>]</p>
    </td>
</tr>
<tr id="abs_Schaub2012a" class="abstract noshow">
    <td><b>Abstract</b>: The Ising model has recently received much attention for the statistical description of neural spike train data. In this paper, we propose and demonstrate its use for building decoders capable of predicting, on a millisecond timescale, the stimulus represented by a pattern of neural activity. After fitting to a training dataset, the Ising decoder can be applied “online” for instantaneous decoding of test data. While such models can be fit exactly using Boltzmann learning, this approach rapidly becomes computationally intractable as neural ensemble size increases. We show that several approaches, including the Thouless–Anderson–Palmer (TAP) mean field approach from statistical physics, and the recently developed Minimum Probability Flow Learning (MPFL) algorithm, can be used for rapid inference of model parameters in large-scale neural ensembles. Use of the Ising model for decoding, unlike other problems such as functional connectivity estimation, requires estimation of the partition function. As this involves summation over all possible responses, this step can be limiting. Mean field approaches avoid this problem by providing an analytical expression for the partition function. We demonstrate these decoding techniques by applying them to simulated neural ensemble responses from a mouse visual cortex model, finding an improvement in decoder performance for a model with heterogeneous as opposed to homogeneous neural tuning and response properties. Our results demonstrate the practicality of using the Ising model to read out, or decode, spatial patterns of activity comprised of many hundreds of neurons.</td>
</tr>
<tr id="bib_Schaub2012a" class="bibtex noshow">
    <td><b>BibTeX</b>:
        <pre>
@article{Schaub2012a,
    author = {Schaub, Michael and Schultz, Simon},
           title = {The Ising decoder: reading out the activity of large neural ensembles},
           journal = {Journal of Computational Neuroscience},
           year = {2012},
           volume = {32},
           pages = {101-118},
           url = {http://arxiv.org/abs/1009.1828},
           doi = {10.1007/s10827-011-0342-z}
}
    </pre></td>
</tr>
</tbody>
</table>
<footer>
 <small>Created by <a href="http://jabref.sourceforge.net">JabRef</a> on 13/11/2018.</small>
</footer>
  <!--END PUBLICATIOON INSERT HERE-->
 </div>
</div>
